src/main/java/com/roommatematcher/enterprise/DAO/IProfileDAO.java

```diff
@@ -0,0 +1,8 @@
+package com.roommatematcher.enterprise.DAO;
+
+public interface IProfileDAO {
+
+    //Need to override to string if you want a string of profiles
+    public String toString();
```

Comment:
toString() method is already defined on java.lang.Object.  No need to redefine it here.